docker/hub/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+cd $(git rev-parse --show-toplevel)
+
+# go run ./cmd/generate-keys --private-key=docker/hub/config/matrix_key.pem -tls-cert=docker/hub/config/server.crt -tls-key=docker/hub/config/server.key
+
+docker build -f docker/hub/Dockerfile -t matrixdotorg/dendrite:latest .
```

Comment:
Hang on, does this mean we're running `docker build` inside a Dockerfile (as `/docker/hub/Dockerfile` runs `RUN sh ./build.sh`) - that sounds a little insane, why are we doing that?

Reply:
We're not. It's running `build.sh` from the root of the repo, not the `docker/hub` folder.